docs-ref-conceptual/azure-cli-configuration.md

```diff
@@ -98,7 +98,7 @@ When you provide a default value, that argument is no longer required by any com
 
 | Section | Name      | Type | Description|
 |---------|-----------|------|------------|
-| __core__ | output | string | The default output format. Can be one of `json`, `jsonc`, `tsv`, or `table`. |
+| __core__ | output | string | The default output format. Allowed values: `json` (default), `jsonc`, `tsv`, `yaml`, `yamlc`, `table`, `none`. |
```

Comment:
The in-tool help shows allowed values alphabetically:

> Allowed values: json, jsonc, none, table, tsv, yaml, yamlc.

I changed the order in the document based on their use frequency.